watchfaces/010-masked-spartan.qml

```diff
@@ -170,4 +189,60 @@ Item {
                 "
         }
     }
+
+    Item {
+        id: nightstandMode
+
+        readonly property bool active: mceCableState.connected //ready || (nightstandEnabled.value && holdoff)
+        //readonly property bool ready: nightstandEnabled.value && mceCableState.connected
```

Comment:
This looks like an error to me?

Reply:
Oh wow, great catch. That is an oversight and should have been chnaged to `nightstand` like in the other WFs.